db/seeding/seed.ts

```
@@ -60,6 +62,14 @@ async function seed() {
     await boardRepo.save(boards);
     console.log('Default board created for test user');
 
+    const boardMembers = boardMemberRepo.create({
+      board: boards[0],
+      userId: testUser[0].id,
+      role: 'owner'
```

ESLint

[error] Insert `,`

(prettier/prettier)